pysat/instruments/madrigal_methods.py

```diff
@@ -238,7 +238,7 @@ def download(date_array, inst_code=None, kindat=None, data_path=None,
                                      '--inst=' + inst_code,
                                      '--kindat=' + kindat])
         print('Feedback from openMadrigal ', a)
-    except OSError as err:
+    except OSError:
```

Comment:
Given @aburrell previous comment about keeping errors, I think we should do the same here. Good to know that pysat had a problem with gloablDownload, even better to have what the system said as well.

Reply:
This was dropped since the err was unused.  adding it back and updating error message.